geomstats/geometry/discrete_curves.py

```diff
@@ -670,7 +671,7 @@ def f_transform_inverse(self, f, starting_point):
             )
         if gs.ndim(f) != gs.ndim(starting_point):
             starting_point = gs.to_ndarray(starting_point, to_ndim=f.ndim, axis=-2)
-        self.l2_metric
+        self.l2_curve_metric
```

Comment:
What does this line do ? I don't see it in the current version of the master branch. For some reason, the current version used to compare changes in this PR does not seem to correspond to the current version of the master branch.

Reply:
This was a left over,thanks for catching it